packages/piro/src/Piro_NOXSolver.hpp

```diff
@@ -97,6 +97,11 @@ class NOXSolver
 
   bool exitUponFailedNOXSolve; 
 
+  enum DerivativeLayout { OP, COL, ROW };
```

Comment:
Can you add doxygen comments to explain the options for DerivativeLayout?

Reply:
Sure. thanks.